avogadro/rendering/solid_first_fs.glsl

```diff
@@ -66,6 +68,47 @@ float lerp(float a, float b, float f)
     return a + f * (b - a);
 }
 
+float rand(vec2 co){
+  return fract(sin(dot(co.xy ,vec2(12.9898,78.233))) * 43758.5453);
+}
```

Comment:
float rand(vec2 co) {
  return fract(sin(dot(co.xy ,vec2(12.9898,78.233)) + cos(dot(co.xy, vec2(93.9898,50.233)))) * 43758.5453);
}
//more complex pattern produced which helps in reduing the noise. @ghutchis your insights ?